docs/configuration.md

```diff
@@ -189,6 +189,16 @@ To use Redis, you need the [PhpRedis](https://github.com/phpredis/phpredis) PHP
 |`AWS_SECRET_ACCESS_KEY`|Secret access key for AWS|
 |`AWS_DEFAULT_REGION`|Default AWS region|
 
+### Authentication Guards
+
+Support for token based authentication like basic auth. Enabled by default.
```

Comment:
Having basic auth enabled in Nginx and using users not existing in Lychee was causing breaking issues for me. The entire site wasn't loading and from what i've figured with a little testing, it was caused by having a user in Nginx that was not existing in Lychee. But it might have been a wrong assumtion as i am not a developer and was just briefly checking where the issue was coming from.

Maybe it would have been easier for me to prevent Nginx from passing through the user and password to Lychee. Even through i am not using the API (unless it's used internally).

I am not sure about the line as it does more then just enabling API support for tokens auth. I don't mind the line to be changed but it is also in [.env.example](https://github.com/LycheeOrg/Lychee/blob/master/.env.example#L63) in Lychee added with my PR you've just passed.

Reply:
That wording was a mistake/oversight :')

Reply:
Would it be possible you change those 2 lines to whatever you see fit?

Reply:
Sorry for causing all this. I can write a new sentence but i am not sure if i might have misunderstood the entire setting. Can you elaborate shortly what people will be missing when disabling the token auth? Does this prevent certain build-in features from working beside API connection for 3rd party?

Reply:
Instead of using the Session interface + login, a user can create an Auth secret token. You can see it as a Bearer token that does not expire. This token can then be used by the interface to authenticate and make direct request to the API e.g. using other tools such as curl, postman etc...

Disabling such functionality has no impact on the functionality of Lychee (aside from not being able to authenticate via tokens).